server/src/schema.graphql

```diff
@@ -17,12 +20,42 @@ type User {
   updatedAt: String!
```

Comment:
there is a thing called " { timestamp: true } " for schemas in mongo db. They automatically create 2 fields for each entry - createdAt and updatedAt of type date. You might be able to leverage it.